src/.nuget/Microsoft.NETCore.Crossgen/runtime.Linux.Microsoft.NETCore.Crossgen.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CpaotCrossgenRuntimeConfigFile>$(ArtifactsObjDir)crossgen2\crossgen.runtimeconfig.json</CpaotCrossgenRuntimeConfigFile>
```

Comment:
Why is this path defined in this file, rather than `Microsoft.NETCore.Crossgen.pkgproj`?

Reply:
No reason. What's the difference?

Reply:
By defining it in each platform-specific props file, the logic is duplicated.

Reply:
Ok. Moved to the pkgproj